code/game/machinery/atmos_exporter.dm

```diff
@@ -0,0 +1,268 @@
+/obj/machinery/atmos_points
+	name = "atmospheric export computer"
+	desc = "used to monitor the galactic atmos markets"
+	icon = 'icons/obj/machines/engi_points.dmi'
+	icon_state = "atmos"
+	verb_say = "states"
+	density = 1
+	anchored = 1
+	var/market_cooldown = 0
+	var/export_target = ""
+	var/export_name = "------"
+	var/list/gas = list()
+	var/list/concentration = list()

```

Comment:
I get flooded by undefined vars every time I do, then Remie will come here and say "put them back" or something reeeee